### PR TITLE
UBERF-8856: Fix space security query and schema update

### PR DESCRIPTION
**Pull request by haiodo**
UBERF-8856: Fix space security query and schema update

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NzU3ZjE4NzE4OWNkMzEwMDNiMDgyNjkiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIn0.bkTt_sE_dvh5IV_YXMN-YHDeuBEVzeef3EDCx_Qywfg">Huly&reg;: <b>UBERF-8875</b></a></sub>

### CODE REVIEW

**Review by BykhovDenis — DISMISSED** [review]

File: server/postgres/src/schemas.ts [hunk 1]
```diff
@@ -152,6 +152,11 @@ const docIndexStateSchema: Schema = {
     type: 'bool',
     notNull: true,
     index: true
+  },
+  objecgClass: {
```
Comment (aonnikov) [inline server/postgres/src/schemas.ts]: typo?

Comment (lexiv0re) [inline server/postgres/src/schemas.ts]: typo object?

**Review by lexiv0re — DISMISSED** [review]

File: server/postgres/src/schemas.ts (hunk 1 above)
Comment (haiodo) [reply server/postgres/src/schemas.ts]: yep

Comment (haiodo) [reply server/postgres/src/schemas.ts]: yep

**Review by aonnikov — APPROVED** [review]

**Pull request merged**